### DISCUSSION TITLE
Lost factory Samples

### DISCUSSION THREAD

**Discussion by StrokeOnGit** [bangcorrupt/hacktribe#319] [Q&A] [answered]
Id like to ask if we could colect some samples and/or if someone would be so kind enough to send me the electribe factory samples

**Comment by bangcorrupt** [comment, marked as answer]
Check out the [Carved Beats Set](https://github.com/bangcorrupt/hacktribe/tree/main/content/carved_beats_set) by @BKLRonin in this repo.

You won't find the original factory samples in this repo, but if you ask on other forums you may have more luck.

Also, if you have the factory sampler, you can install the factory firmware and export the samples.  They are still in the flash memory.

See [Factory Samples](https://github.com/bangcorrupt/hacktribe/wiki/How-To#factory-samples) in the wiki.

**Comment by StrokeOnGit** [comment]
I have been trying to get the PCM samples for a couple of days now, I have the synth with Hacktribe already installed do I need to modify the hadder back to the original for the dump to work or am I missing something. I have close to no coding experience and not being a native speaker certainly won't help either, thanks in advance!

**Reply by bangcorrupt** [reply]
To dump the PCM samples from the synth hardware, you need to patch the sampler firmware using the `synth-pcm-dump` branch.  Once you install that, you can use the [dump_extracted_pcm.py](https://github.com/bangcorrupt/hacktribe/blob/synth-pcm-dump/scripts/dump_extracted_pcm.py) script to get the PCM data as a single block.  Then reinstall Hacktribe patched from the `main` or `experimental` branch.

https://github.com/bangcorrupt/hacktribe/wiki/Dump-Synth-PCM-Samples

https://github.com/bangcorrupt/hacktribe-tutorial/issues/2
